llvm-external-projects/iree-dialects/include/iree-dialects/Dialect/VectorExt/IR/VectorExtOps.h

```diff
@@ -70,27 +69,70 @@ class DimensionalRange {
 // required during distribution.
 class LayoutIterator {
```

Comment:
Can we move this out of VectorExtOps.h into it's own file?

Reply:
Sure, but what's the reason for doing this? There's nothing in VectorExtOps.h except for the LayoutIterator class and classes that it uses.

Reply:
It isn't really related to VectorExtOps at all. But let's do it in a seperate patch.

Reply:
Will do thanks